cra-tf-validate-ignore-goals.json

```diff
@@ -23,6 +23,12 @@
             "description:": "Check whether Kubernetes Service clusters are enabled with IBM Cloud Monitoring",
             "ignore_reason": "False alarm. In order to have more control over the way observability agents are configured and deployed, a helm chart installation is used rather than the native provider support.",
             "is_valid": false
+        },
+        {
+            "scc_goal_id": "3000907",
+            "description:": "Check whether OpenShift version is up-to-date",
+            "ignore_reason": "Tracking in https://github.ibm.com/GoldenEye/issues/issues/4000",
+            "is_valid": true
```

Comment:
The fix for this is coming today - I'd rather we didnt ignore the goal and wait for fix

Reply:
Yea, sure. I added it to pass the cra scan and check if the PR pipeline runs fine.